test/integration/test_deepgram_live.py

```
@@ -0,0 +1,50 @@
+def base_url():
+    from ovos_stt_http_server.routers.deepgram import make_deepgram_router
+
+    def register(app, model):
+        app.include_router(make_deepgram_router(model))
+
+    yield from run_live_server(register)
+
+
+def test_listen_basic(base_url):
+    wav = make_silent_wav()
+    r = requests.post(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)